### PR TITLE
Support Azure Blob Storage on AKS via access key

### PR DESCRIPTION
**Pull request by AdheipSingh**
Fixes #XXXX.

### Description

<hr>

This PR has:
- [x] been tested to ensure log ingestion and log query works.
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added documentation for new or modified features or behaviors.

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ parseable:
   ## Set to true if you want to deploy Parseable in local mode (store logs
   ## on local mount point instead of S3 bucket)
   local: false
+  ## object store can be s3, blob
+  ## local needs to be false if set to object store.
+  objectStore: s3-store
```
Comment (nitisht) [inline helm/values.yaml:11]: Shall we instead change it to 

```
store: local / s3-store / blob-store
```

**Comment by AdheipSingh** [comment]
tested on azure.

File: helm/templates/querier-statefulset.yaml [hunk 2]
```diff
@@ -60,11 +65,6 @@ spec:
               secretKeyRef:
                 name: {{ $secret.name }}
                 key: {{ $key }}
-          - name: HOSTNAME
-            valueFrom:
-              fieldRef:
-                apiVersion: v1
-                fieldPath: metadata.name
```
Comment (nitisht) [inline helm/templates/querier-statefulset.yaml:67]: This is needed, why remove this?

Comment (AdheipSingh) [reply helm/templates/querier-statefulset.yaml:67]: https://github.com/AdheipSingh/parseable/blob/aks-helm/helm/templates/querier-statefulset.yaml#L50
This is present just refactored it

Comment (nitisht) [reply helm/templates/querier-statefulset.yaml:67]: oh right, my bad.

**Pull request merged**